src/tools/kdb/specmount.cpp

```diff
@@ -83,7 +83,7 @@ void SpecMountCommand::buildBackend (Cmdline const & cl)
 
 	if (!cl.quiet && backends.empty ())
 	{
-		std::cout << "No metadata \"mountpoint\" found below spec" << mp << std::endl;
+		std::cout << "No metadata \"mountpoint\" found on key \"spec" << mp << "\"" << std::endl;
```

Comment:
Does this really print the whole key correctly, or does it just fix the quotes?
According to the issue it should print like this:
```
No metadata "mountpoint" found on key "spec:/tests/example"
```
Note the colon `:` symbol between namespace and the key name.

Reply:
good catch!